src/components/Firebase/Firebase.tsx

```diff
@@ -20,14 +21,22 @@ type FirebaseProviderProps = {
     appId: ConfigVariableValue;
     measurementId: ConfigVariableValue;
   };
+  renderLoading?: React.ReactNode;
```

Comment:
Prop got me confused for a sec since it sounded like a boolean prop.

Reply:
Happy to rename it. How do you feel about `renderLoader`?

Reply:
What do you think about `loaderNode`?